### DISCUSSION TITLE
请求访问ROSIE软件

### DISCUSSION THREAD

**Discussion by Zhouziqian1** [RosettaCommons/rosetta#84] [ROSIE]
Hello,
I'd like to access ROSIE but I'm getting an error that I have to confirm my institution is academic. I currently study at Institute of Zoology, Chinese Academy of Sciences (http://www.ioz.ac.cn/). Thus, it would be great if both got marked as academic institutions.
Thank you and have a nice day,
Ziqian Zhou

**Comment by lyskov** [comment]
Thank you for letting us know! Could you please let me know which domain name your emails have so i can add it? Is it `@.ioz.ac.cn`? Thanks,

**Reply by Zhouziqian1** [reply]
Yes, thank you.

**Comment by lyskov** [comment]
This should be fixed now. Please feel free to to re-open if you still having issues during login.
